src/spectro_inlets_quantification/sensitivity.py

```diff
@@ -1137,3 +1140,59 @@ def plot_F_vs_f(
             F_fit = self.alpha * f_fit
             ax.plot(f_fit, F_fit, "k--")  # type: ignore
         return ax
+
+
+def append_sensitivity_factors(*args):
+    """Recursively append elements in a list"""
+    sf = append_two_sensitivity_factors(args[0], args[1])
+    if len(args) == 2:
+        return sf
+    remaining_args = [sf] + list(args[2:])
+    return append_sensitivity_factors(*remaining_args)
+
+
+def append_two_sensitivity_factors(
+        sf_1: Union[SensitivityFactor, SensitivityList],
```

Comment:
If you find this sort of type definition unhelpful for the reading experience, you can consider defining a TypeAlias by which you give a custom type, like a this case a Union, a name of its own. In this case it could be something like "SensitivityLike" or similar, to indicate that the type is is something that behaves like something or can be used as something. You can see an example of this in the types for gases, where I made the MixtureLike, which I think is an alias for the string/dict/Miuxture that a lot of methods accept.

Reply:
I find it fine for the reading process, but agree with what we talked about with a SensitivityLike base class... but I didn't understand the implementation difference between that and a mix-in class, so I didn't touch it for now.

Reply:
I think the two discussions are separate. This comment was **only** about the type definitions.